src/plugins/imagery/components/ImageryView.vue

```diff
@@ -411,8 +411,11 @@ export default {
     watch: {
         imageHistorySize(newSize, oldSize) {
             let imageIndex;
-            if (this.indexForFocusedImage !== undefined) {
-                imageIndex = this.initFocusedImageIndex;
+            if (this.focusedImageTimestamp !== undefined) {
```

Comment:
Image history size shouldn't change when `focusedImageIndex` is provided since it's paused.
Maybe we can optimize later. I feel like it's good to check it everytime because the index might go out of bounds.